lib/websocket2.0.js

```diff
@@ -116,6 +121,11 @@ let WebSocketV2 = function (params) {
 			) {
 				throw new Error('Invalid Exchange type pased');
 			}
+
+			if (mode === MODE.Depth && exchangeType !== EXCHANGES.nse_cm) {
+				throw new Error("Depth 20 only supports NSE_CM");
```

Comment:
Correct the sentence Depth 20 only supports NSE_CM.

Reply:
Sentence has been changed.